python/TestHarness/TestHarness.py

```diff
@@ -738,6 +738,7 @@ def parseCLArgs(self, argv):
     parser.add_argument('--max-fails', nargs=1, type=int, dest='max_fails', default=50, help='The number of tests allowed to fail before any additional tests will run')
     parser.add_argument('--pbs', nargs='?', metavar='batch_file', dest='pbs', const='generate', help='Enable launching tests via PBS. If no batch file is specified one will be created for you')
     parser.add_argument('--pbs-cleanup', nargs=1, metavar='batch_file', help='Clean up the directories/files created by PBS. You must supply the same batch_file used to launch PBS.')
+    parser.add_argument('--pbs-project', nargs='?', default='moose', help='Identify PBS job submission to specified project')
```

Comment:
@milljm  Just to confirm:
If I leave this option out it'll default to "moose", however if I pass this option, I'm also required to pass an argument, correct?